buildSrc/src/main/kotlin/kotlin-multiplatform-library-convention.gradle.kts

```diff
@@ -0,0 +1,72 @@
+@file:Suppress("UnstableApiUsage")
+
+plugins {
+  id("org.jetbrains.kotlin.multiplatform")
+  id("org.jetbrains.compose")
+  id("com.android.library")
+}
+
+kotlin {
+  android {
+    publishLibraryVariants("release")
+    compilations.configureEach {
+      kotlinOptions {
+        jvmTarget = "11"
+      }
+    }
+  }
+
+  jvm("desktop") {
+    compilations.configureEach {
+      kotlinOptions {
+        jvmTarget = "11"
+      }
+    }
+  }
+
+  // TODO what other targets?
+
+  targets.configureEach {
+    compilations.configureEach {
+      compilerOptions.configure {
+        freeCompilerArgs.addAll(
+          "-Xcontext-receivers",
+          "-Xjvm-default=all", // TODO IDE complains this isn't set (transformableState.kt). is it working?
```

Comment:
I've asked on kotlinlang slack: https://slack-chats.kotlinlang.org/t/12027617/wave-is-anyone-using-xjvm-default-all-in-their-project-this-#70174f9f-d1ff-4a44-823e-4258f80e4a6f

Reply:
wanna try out @jakewharton's recommendation?

https://github.com/cashapp/redwood/blob/9082d4182f9622b95dfd3fd1bc2dcf558ff11d31/build.gradle#L91-L98

Reply:
No change, IDE still complains.

Reply:
Sigh okay. Let's get this PR merged for now. We can continue looking for a fix afterwards.